Firestore/core/src/core/query.h

```diff
@@ -118,16 +118,11 @@ class Query {
   const model::FieldPath* InequalityFilterField() const;
 
   /**
-   * Returns the first array operator (array-contains or array-contains-any)
-   * found on a filter, or absl::nullopt if there are no array operators.
+   * Checks if any of the provided filter operators are included in the query
+   * and returns the first one that is, or null if none are.
    */
-  absl::optional<Filter::Operator> FirstArrayOperator() const;
-
-  /**
-   * Returns the first disjunctive operator (IN or array-contains-any) found
-   * on a filter, or absl::nullopt if there are no disjunctive operators.
-   */
-  absl::optional<Filter::Operator> FirstDisjunctiveOperator() const;
+  absl::optional<Filter::Operator> FindOperator(
+      std::vector<Filter::Operator> ops) const;
```

Comment:
Since `FindOperator` isn't storing the `ops` value in a field (or otherwise retaining it beyond the lifetime of the call), you should pass `ops` via const reference.